RoadMap_SwiftUI/RoadMap_SwiftUI/ViewModel/FilterViewModel.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  FilterViewModel.swift
+//  RoadMap_SwiftUI
+//
+//  Created by Артур Щукин on 27.01.2023.
+//
```

Comment:
```suggestion
//
//  FilterViewModel.swift
//  RoadMap_SwiftUI
//
//  Created by Артур Щукин on 27.01.2023.
//
```
не отработал форматер